Distraction Shield/optionsPage/connectDataToHtml.js

```diff
@@ -3,32 +3,33 @@
  * to the actual html representations of this data.
  */
 
-sortListOnCheckboxVal = function(bsA, bsB) {
-    var valueOfA = bsA.getCheckboxVal();
-    var valueOfB = bsB.getCheckboxVal();
+export function sortListOnCheckboxVal(bsA, bsB) {
```

Comment:
this function does not do the sorting, it's a comparator of some sort, thus the naming is misleading